iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ReactiveCapabilityCurve.java

```diff
@@ -70,4 +70,15 @@ public interface Point {
      */
     double getMaxP();
 
+    /**
+     * Get the reactive power minimum value of the curve (with the possibility of extrapolating slope of reactive
+     * limits outside active limits)
+     */
+    double getMinQ(double p, boolean extrapolateReactiveLimitSlope);
+
```

Comment:
You should mark the PR as breaking change because the netork store implementation will have to implement this new methdo.